static/sass/_pattern_blog-card.scss

```diff
@@ -0,0 +1,132 @@
+@mixin blog-p-card {
+  .blog-p-card__category {
```

Comment:
Would it be worth using the vanilla blog card style? https://vanillaframework.io/docs/patterns/card#content-bleed

Reply:
Didn't use `content-bleed`, just kept the default blog styling, because the newsletter has padding and it will feel out of place.

Reply:
If we are building something new, it is best to use the Vanilla pattern. This is the purpose of Vanilla.